app/main/util/email_verification.py

```diff
@@ -0,0 +1,20 @@
+from flask import current_app
+from itsdangerous import URLSafeTimedSerializer
+
+
+def generate_confirmation_token(email):
+    serializer = URLSafeTimedSerializer(app.config['SECRET_KEY'])
```

Comment:
You've imported current_app but used app.

Reply:
I wrote import current_app as app. this should work, right?